packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -303,7 +309,11 @@ module.exports = {
 
     // require or disallow space before function opening parenthesis
     // http://eslint.org/docs/rules/space-before-function-paren
-    'space-before-function-paren': ['error', { anonymous: 'always', named: 'never' }],
+    'space-before-function-paren': ['error', {
+      anonymous: 'always',
+      named: 'never',
+      asyncArrow: 'always'
```

Comment:
this needs to stay `ignore` since eslint 3.7 parses `async () => {}` just fine, so `always` would be a breaking change.